br/pkg/restore/snap_client/systable_restore.go

```diff
@@ -145,6 +145,35 @@ func isUnrecoverableTable(schemaName string, tableName string) bool {
 	return ok
 }
 
+func IsStatsTemporaryTable(tempSchemaName, tableName string) bool {
+	_, ok := GetDBNameIfStatsTemporaryTable(tempSchemaName, tableName)
+	return ok
+}
+
+func GetDBNameIfStatsTemporaryTable(tempSchemaName, tableName string) (string, bool) {
+	if name, ok := utils.StripTempDBPrefixIfNeeded(tempSchemaName); ok && isStatsTable(name, tableName) {
+		return name, true
+	}
+	return "", false
+}
+
+func GenerateMoveStatsTableSQLPair(statisticTables map[string][]string) (renameSQL, dropSQL string) {
```

Comment:
Did you miss support to other system tables of privilege tables?

Reply:
Done. Currently only support privilege tables and `bind_info`, which are recorded in `renameableTables`.